package/upgrade/upgrade_node.sh

```diff
@@ -5,6 +5,15 @@ SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 source $SCRIPT_DIR/lib.sh
 UPGRADE_TMP_DIR=$HOST_DIR/usr/local/upgrade_tmp
 
+trap clean_up_tmp_files EXIT
+
+clean_up_tmp_files()
+{
+  echo "Clean up large tmp files..."
+  \rm -vf "$NEW_OS_SQUASHFS_IMAGE_FILE"
+  \rm -vf "$tmp_rootfs_squashfs"
```

Comment:
echo "Clean up large tmp files..." -> echo "Clean up tmp files..."   

should L#433 `umount $tmp_rootfs_mount` also be executed here? to umount first, then delete

and, is it better to catch `elemental upgrade` return code first, check return code, echo, and finally clean up ?

Reply:
another point:

should the `upgrade_node.sh` call `cat /usr/local/upgrade_tmp/elemental-upgrade-20221103093405.log` when `elemental upgrade` return code is not 0 ? thus `kubectl get log .... ` will have the error log directly